react/week-19/lesson.md

```diff
@@ -189,19 +188,15 @@ There are 3 important parts in this code:
 
 #### Component Composition
 
-A component can be combined with another component so that both are rendered. This is sometimes called *composition* ([interactive example](https://codesandbox.io/s/0x4wonqn00)):
+A component can be combined with another component so that both are rendered. This is called *composition* ([interactive example](https://codesandbox.io/s/0x4wonqn00)):
```

Comment:
I can get these codesandboxes updated with those tweaks.

I suppose we could probably set up a group account or something so that they're not all under my account.

Also, while I'm on the subject - the codesandboxes are all configured to be "frozen", which means that if you edit them, and accidentally hit save then it will prompt you to fork. I find it **really** useful to be able to open up live examples and play around with them, but I hit save out of habit so now I won't mess up the original example :)